code/ATMOSPHERICS/components/unary_devices/cryo.dm

```diff
@@ -203,22 +203,18 @@
 		if(!state_open)
 			on = 1
 
-	if(href_list["open"])
-		open_machine()
-
-	if(href_list["close"])
-		if(close_machine() == usr)
-			var/datum/nanoui/ui = SSnano.get_open_ui(usr, src, "main")
-			ui.close()
-			on = 1
 	if(href_list["switchOff"])
 		on = 0
 
+	if(href_list["ejectOccupant"])

```

Comment:
Yep, checked it and it works great. Only tested with a mindless `/mob/living/carbon/human` though.